docs/docsite/rst/user_guide/intro_bsd.rst

```diff
@@ -26,7 +26,7 @@ Bootstrapping BSD
 
 Ansible is agentless by default, however, it requires Python on managed nodes. Only the :ref:`raw <raw_module>` module will operate without Python. Although this module can be used to bootstrap Ansible and install Python on BSD variants (see below), it is very limited and the use of Python is required to make full use of Ansible's features.
 
-The following example installs Python 2.7 which includes the json library required for full functionality of Ansible.
+The following example installs Python which includes the json library required for full functionality of Ansible.
```

Comment:
```suggestion
The following example installs Python 3 which includes the JSON library required for the full functionality of Ansible.
```

Reply:
Since the FreeBSD part still uses 2.7, I'd rather go without mentioning the major version.

Reply:
OK.